pyvista/core/pointset.py

```diff
@@ -1062,11 +1068,36 @@ def n_strips(self) -> int:  # numpydoc ignore=RT01
         """
         return self.GetNumberOfStrips()
 
+    @staticmethod
+    def use_strict_n_faces(mode: bool) -> None:
+        """Global opt-in to strict n_faces.
+
+        Parameters
+        ----------
+        mode : bool
+            If true, all future calls to :attr:`n_faces <pyvista.PolyData.n_faces>`
+            will return the same thing as :attr:`n_faces_strict <pyvista.PolyData.n_faces_strict>`.
+
+        """
+        global _USE_STRICT_N_FACES
+        _USE_STRICT_N_FACES = mode
+
```

Comment:
In my mind `use_strict_n_faces` should be a function. This is because this operation affects all objects in the library.